src/v2/Components/v2/ArtworkFilter/__tests__/ArtworkFilterContext.jest.tsx

```diff
@@ -129,28 +95,141 @@ describe("ArtworkFilterContext", () => {
       act(() => {
         expect(context.filters.page).toEqual(10)
         context.unsetFilter("sort")
-        setTimeout(() => {
-          expect(context.filters.page).toEqual(1)
-          done()
-        })
       })
+      expect(context.filters.page).toEqual(1)
     })
 
-    it("#resetFilters", () => {
-      getWrapper({
-        filters: {
+    describe("staged filter changes", () => {
```

Comment:
Verbose testing block here, but in a nutshell this all supports the stage filter updates —

![Screen Shot 2020-06-04 at 3 56 12 PM](https://user-images.githubusercontent.com/140521/83804342-fa5fec00-a67b-11ea-8cac-cdfa2a70fe06.png)